DDF for Tuya presence sensor (_TZ3000_bb6xaihh with model id SNZB-03)

Doskii opened the pull request:
This DDF works for the ["Smart PIR Motion Sensor" I got from AliExpress](https://www.aliexpress.com/item/1005008787969969.html). It is being reported as "_TZ3000_bb6xaihh" but with the model id "SNZB-03" instead of the previously supported "TS0202".

The config is very similar to the [TS0202](https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/tuya/ts0202_presence_sensor.json) but I was not sure if I should have added the "SNZB-03" to that DDF or not as the name of the file would no longer make sense.

![package](https://github.com/user-attachments/assets/1d2ddc3c-70f0-4421-b2c2-12b1c521faba)

![deCONZ](https://github.com/user-attachments/assets/7c0f6bc2-f7b0-4678-9d6d-c77a602f1926)

Smanar commented:
Honnestly for me you can just add it to the existing DDF, even the file name was not the good one.
Not the first time we can see that, for exemple > https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/tuya/_TZ3000_TS0203_door_sensor.json (look at the second one).

Doskii commented:
@Smanar Ok I've removed my DDF, added the module to `tuya/ts0202_presence_sensor.json` and tested it again.

Though I did notice that when I thought `_TZ3000_bb6xaihh` was already in there but with `TS0202` instead I was apparently slightly off and was looking at `_TZ3040_bb6xaihh` instead (notice the 40). But it works nonetheless and is basically the same anyways so should be good.

Smanar commented:
Yeah tuya magic ...
IDK if they have not finished to change the modelID on the firwmare or if it's normal
